feat: multilobby support

0xMugen opened the pull request:
multi lobby support

guiltygyoza commented:
Attached is a screenshot from the vercel preview. The input box is confusing to me as in I'm not sure if it's for username of lobby name. I would have expected two separate input boxes, one for username, the other for lobby name
<img width="1440" alt="Screenshot 2024-05-19 at 1 51 01 PM" src="https://github.com/topology-gg/esotere-client/assets/59590480/53221387-eb80-4459-a2ab-6b59d4561ab7">

0xMugen commented:
- added details for what input box does what.
- temp fix: refreshes the page when Aspect ratio is changed. 
- removed the chat box
- added emotes as reactions with buttons

d-roak commented:
I'm merging this to author another PR based on this one
